app/src/main/java/org/systers/mentorship/view/activities/RequestDetailActivity.kt

```diff
@@ -134,14 +137,17 @@ class RequestDetailActivity: BaseActivity() {
         }
     }
 
-    private fun setObservables() {
+    private fun setObservables(relationResponse: Relationship) {
         requestDetailViewModel  = ViewModelProviders.of(this).get(RequestDetailViewModel::class.java)
         requestDetailViewModel.successful.observe(this, Observer {
             successful ->
             hideProgressDialog()
             if (successful != null) {
                 if (successful) {
                     Toast.makeText(this, requestDetailViewModel.message, Toast.LENGTH_LONG).show()
+                    val previousScreen = Intent(applicationContext, RequestPagerFragment::class.java)
+                    previousScreen.putExtra("requestid", relationResponse.id.toString())
+                    setResult(1000, previousScreen)
```

Comment:
Can you create a constant for this 1000 value, and then call it both on this file and the other file.
This should not be hardcoded.
Maybe call it DELETE_REQUEST_RESULT_ID?!

@m-murad